ts/Dashboards/Actions/Bindings.ts

```diff
@@ -95,6 +95,7 @@ namespace Bindings {
         const optionsStates = (options as any).states;
         const optionsEvents = options.events;
         const renderTo = options.renderTo || options.cell;
+        const currentBoard = board || cell?.row?.layout.board;
```

Comment:
Wouldn't it be easier to have a `board` as a required argument? We will get rid of a lot of question marks and checks.